### PR TITLE
Andsemenov/footer add hyperlinks

### PR DESCRIPTION
**Pull request by andsemenov**

### CODE REVIEW

**Review by dbxnr — CHANGES_REQUESTED** [review]
Looks good, although can you properly capitalise the alt text? e.g. 'Traverse Theatre logo'

**Comment by andsemenov** [comment]
I have capitalised logo's alt attribites, could you check again, please. Thank you.

File: index.html [hunk 1]
```diff
@@ -830,26 +830,38 @@ <h2 class="blog-item-title font-alt text-center">Everything is the 'new normal'<
...
+                    <img id="logo3" src="./images/footer/Logo-Traverse-Theatre.png" alt="Traverse Theatre logo">
+                  </a>
+                  <a class="footer-logo" href="https://www.cca-glasgow.com/programme#exhibitions" target="_blank">
+                    <img id="logo4" src="./images/footer/Logo-CCA.png" alt="CCA logo">
+                  </a>
+                  <a class="footer-logo" href="http://www.macarts.scot/" target="_blank">
+                    <img id="logo5" src="./images/footer/Logo-MacArts.png" alt="MacArts logo">
+                  </a>
+                  <a class="footer-logo" href="https://www.aberdeenartscentre.com/" target="_blank">
+                    <img id="logo6" src="./images/footer/Logo-Aberdeen-Arts-Logo.png" alt="Aberdeen Arts logo">
```
Comment (dbxnr) [inline index.html]: Should be Aberdeen Arts Center

File: index.html [hunk 2]
```diff
@@ -830,26 +830,38 @@ <h2 class="blog-item-title font-alt text-center">Everything is the 'new normal'<
...
+                  </a>
+                  <a class="footer-logo" href="https://www.tramway.org/Pages/home.aspx" target="_blank">
+                    <img id="logo7" src="./images/footer/Logo-Tramway.png" alt="Tramway logo">
+                  </a>
+                  <a class="footer-logo" href="https://lanntair.com/" target="_blank">
+                    <img id="logo8" src="./images/footer/Logo-An-Lanntair.png" alt="Lanntair logo">
+                  </a>
+                  <a class="footer-logo" href="https://www.horsecross.co.uk/" target="_blank">
+                    <img id="logo9" src="./images/footer/Logo-Perth-X-Theatre.png" alt="Perth X Theatre logo">
```
Comment (dbxnr) [inline index.html]: Should be Perth Theatre

File: index.html [hunk 3]
```diff
@@ -830,26 +830,38 @@ <h2 class="blog-item-title font-alt text-center">Everything is the 'new normal'<
 
                 <!-- Logos -->
 
-
+                Traverse Theatre logo
```
Comment (dbxnr) [inline index.html]: I think this is a mistake? :)

File: index.html (hunk 2 above)
Comment (andsemenov) [reply index.html]: Done. Thanks.

File: index.html (hunk 1 above)
Comment (andsemenov) [reply index.html]: Done (Centre). Thanks.

File: index.html (hunk 3 above)
Comment (andsemenov) [reply index.html]: Sorry, my mistake. Corrected, thanks.

**Review by dbxnr — APPROVED** [review]

**Pull request merged**